corehq/apps/reports_core/static/reports_core/js/charts.js

```diff
@@ -165,7 +165,7 @@ var charts = (function() {
                 }
                 var $svg = d3.select(chartContainer[0]).append("svg");
                 var id = 'chart-' + i;
-                $svg.attr({id: id, width: "50%", height: "200"});
+                $svg.attr({id: id, height: "400"});
```

Comment:
Awesome. Was 400 arbitrary? Might be worth checking out/designing for how
it looks on our most popular resolution: 1366x768

On Mon, Feb 22, 2016 at 3:37 PM, Ben <notifications@github.com> wrote:

> In corehq/apps/reports_core/static/reports_core/js/charts.js
> <https://github.com/dimagi/commcare-hq/pull/10419#discussion_r53688773>:
>
> > @@ -165,7 +165,7 @@ var charts = (function() {
> >                  }
> >                  var $svg = d3.select(chartContainer[0]).append("svg");
> >                  var id = 'chart-' + i;
> > -                $svg.attr({id: id, width: "50%", height: "200"});
> > +                $svg.attr({id: id, height: "400"});
>
> i don't think the width attribute actually did anything here. @czue
> <https://github.com/czue> was there something you were trying to
> accomplish with that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/10419/files#r53688773>.
>